LocalSettings.php

```diff
@@ -260,6 +260,10 @@
 	 ],
 
 	// CreateWiki
+	'wgCreateWikiDatabaseClusters' => [
+		'db4',
```

Comment:
db4 and db5 are not clusters, c1 and c2 are

Reply:
Thanks, done.